.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "eslint.enable": false
+}
```

Comment:
Maybe you need to keep this file to .gitignore?

Reply:
add .gitignore